druid-shell/Cargo.toml

```diff
@@ -59,16 +48,19 @@ core-graphics = "0.22.0"
 foreign-types = "0.3.2"
 bitflags = "1.2.1"
 
-# TODO(x11/dependencies): only use feature "xcb" if using X11
 [target.'cfg(target_os="linux")'.dependencies]
-cairo-rs = { version = "0.9.1", default_features = false, features = ["xcb"] }
-gio = "0.9.1"
-gdk = "0.13.2"
-gdk-sys = "0.10.0"
-glib = "0.10.1"
-glib-sys = "0.10.0"
-gtk-sys = "0.10.0"
-gtk = { version = "0.9.2", features = ["v3_22"] }
+# TODO(x11/dependencies): only use feature "xcb" if using X11
+cairo-rs = { version = "0.9.1", default_features = false, features = ["xcb"], optional = true }
```

Comment:
we currently allow cairo to be used, opt-in, on macOS. I'm not sure we should, but we do; I think this is why these deps are in two places.

Reply:
Huh I did not know that. I'll move Cairo back to the "normal" dependencies then.

Reply:
This proved to be a bit tricky, but it should work now.